README.md

```diff
@@ -11,8 +11,8 @@
 
 1. Make sure you have [composer](https://getcomposer.org/) installed.
 2. Run `composer create-project forkcms/forkcms .` in your document root.
-3. Browse to your website
-4. Follow the steps on-screen
+3. Browse to your website.
+4. Follow the steps on-screen.
```

Comment:
This is actually more consistent 😁

Reply:
it is, but I don't agree with replacing the talk to
And look at other 3 prs
Feel free to create one to fix the .'s wouter :p